components/header/avatar.tsx

```
@@ -0,0 +1,46 @@
+  className?: string;
+}
+
+const sizeMap = {
+  sm: "h-8 w-8 text-xs",
+  md: "h-10 w-10 text-sm",
+  lg: "h-12 w-12 text-base",
+};
+
+export function Avatar({ src, alt = "", fallback, size = "sm", className }: AvatarProps) {
+  const [imageError, setImageError] = useState(false);
+  const dimensions = parseInt(sizeMap[size].split(" ")[0].replace("h-", "")) * 4; // Convert tailwind sizes to pixels
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)